app/models/manageiq/providers/kubevirt/infra_manager/vm/operations.rb

```diff
@@ -21,16 +21,16 @@ module ManageIQ::Providers::Kubevirt::InfraManager::Vm::Operations
 
   def raw_destroy
     ext_management_system.with_provider_connection do |connection|
-      # Retrieve the details of the offline virtual machine:
+      # Retrieve the details of the virtual machine:
       begin
-        live_vm = connection.live_vm(name)
+        vm_instance = connection.vm_instance(name)
       rescue KubeException
```

Comment:
Maybe not in this PR but we need to remove KubeException here. I wonder whether it would be worth to run kubeclient exception in fog-kubevirt specific exception.

Reply:
@pkliczewski we actually have a card to track that task.

Reply:
@pkliczewski I'd take this one step further and eliminate the dependency of miq-providers-kubevirt on kubeclient. I think that we shouldn't have a direct dependency on kubeclient at all.

Reply:
:+1: